src/main/java/com/jfb/tickets/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package com.jfb.tickets.controller;
+
+import com.jfb.tickets.model.Ticket;
+import com.jfb.tickets.model.User;
+import com.jfb.tickets.service.UserService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("")
+    public User createUser(@RequestBody User user) {
+        return userService.saveUser(user);
+    }
+
+    @GetMapping("")
+    public List<User> getAllUsers() {
+        return userService.getUsers();
+    }
+
+    @GetMapping("/{userId}")
+    public User getUserById(@PathVariable UUID userId) {
+        Optional<User> userOptional = userService.getUserById(userId);
+
+        return userOptional.orElse(null);
```

Comment:
**required**:
1. it is better do not use on controller level come logic. you can write just:.
`return userService.getUserById(userId);`

2. Logic related to null case move to service. 
Instead of `orElse` use `orElseThrow` with throwing custom or any other exception. 

**optionally**:
This exception will be displayed in log, and f you want to return specific error response to your client, read about `@ExceptionHandler` and `@ControllerAdvice`.